src/config.rs

```diff
@@ -22,6 +22,7 @@ pub struct Config {
     pub description: Option<String>,
     pub link: Option<String>,
     pub ignore: Vec<Pattern>,
+    pub include: Vec<Pattern>,
```

Comment:
Please give more detail on this change in your commit message.  They are an important place to communicate.

For example, some useful things to include
- Link to #221 to help give context since sometimes designs are worked out in issues
- Especially if the user-facing design isn't fleshed out in the issue, describe it (the What?).  For example "To allow overriding cobalt's default ignore behavior, this adds a new `include` field to cobalt.yml that ...".
- On related note, be sure to include how this feature interacts with other features.  How do `ignore` and `include` interact? Should it be defined or undefined behavior?  What should it be defined to?  Why?
- "Why?" on your implementation, if any is needed, is also important.  This is a simple enough change, I don't expect much of a "Why?" regarding the implementation.

Notice that I separate out user-facing design / workflow separate from implementation.  "How?" with implementation isn't as important; the code covers that.